ompi/mca/osc/ucx/osc_ucx_component.c

```diff
@@ -716,6 +717,18 @@ static int component_select(struct ompi_win_t *win, void **base, size_t size, in
         if (ret != OMPI_SUCCESS) {
             goto error;
         }
+    } else if (flavor == MPI_WIN_FLAVOR_DYNAMIC) {
+
+        ret = opal_common_ucx_wpmem_create(module->ctx, &dynamic_base, 0,
```

Comment:
Can this be included in the above branch and use `module->mem`? I don''t see why we need to keep two mem objects when only one is ever used...

Reply:
I think we should. I'll work on it